crates/forge/tests/integration/cheat_sequencer_address.rs

```diff
@@ -246,7 +246,7 @@ fn cheat_sequencer_address_with_span() {
 
                 let target_sequencer_address: ContractAddress = 123.try_into().unwrap();
 
-                cheat_sequencer_address(dispatcher.contract_address, target_sequencer_address, CheatSpan::TargetCalls(1));
+                cheat_sequencer_address(dispatcher.contract_address, target_sequencer_address, CheatSpan::target_calls(1));
```

Comment:
This will not compile/work. There is no method `CheatSpan::target_calls` in the `cheatcodes.cairo`.